subsys/bluetooth/host/mesh/cfg_srv.c

```diff
@@ -42,6 +43,7 @@ static struct bt_mesh_cfg_srv *conf;
 static struct label {
 	u16_t addr;
 	u8_t  uuid[16];
+	atomic_t ref;
```

Comment:
I'd have gone with u16_t in the beginning of the struct, which increases it by two bytes. Adding atomic_t at the end increases it by 6 bytes, and we don't need the 32-bit range or atomicity here (due to the context types that this gets accessed in)